src/app/downloads/chart/dropday-chart-presentation.component.css

```diff
@@ -0,0 +1,13 @@
+:host {
+  display: block;
+  background-color: white;
+}
+
+/* don't display more than two lines of text on category chart labels (episode names) */
```

Comment:
I truncated the labels because C3 breaks it into separate lines to try to fit it all in there, still didn't want to show any more than two lines in case it does weird text breaking. Definitely an imperfect workaround to getting episode names to fit.